oximeter/oximeter/src/histogram.rs

```diff
@@ -180,6 +181,7 @@ where
 
 /// Type storing bin edges and a count of samples within it.
 #[derive(Debug, Clone, Copy, PartialEq, Deserialize, Serialize, JsonSchema)]
+#[schemars(rename = "Bin{T}")]
```

Comment:
I think this is an example of what you referred to in your comment. It's not great that this is generated as `Bindouble`, instead of e.g., `BinDouble`. But it's also internal, which means I don't think anyone should see it when writing code.

If I understand your point about "curated types", you mean something like creating a type alias `pub type BinDouble = Bin<f64>;`, and the _former_ is what appears in the API. Is that right?

Reply:
Yes. Exactly!